xs2a-adapter-rest-impl/src/main/resources/static/index.html

```diff
@@ -126,12 +157,26 @@ <h1 class="registry-header">Aspsp Registry Manager</h1>
         </table>
     </div>
     <div class="welcome-message">
-        <h1>Hey! There is nothing to display right now :(</h1>
+        <h1>Adapter team wishes you good luck!<br> Nothing to display right now.</h1>
```

Comment:
Good message, but let's write something like that "to start working with ASPSP registry try to search for something or add new record". IMHO it would be more helpfull

Reply:
got it

Reply:
done